UPGRADE.md

```diff
@@ -53,12 +94,13 @@ The Learning Material migration *copies* learning materials, it does not *move*
 
 To find out how much space your learning materials are currently taking, login to your Ilios server and change directories to the folder that contains your learning materials and then run this command:
 
-```bash
+ ```bash
 df -h . 
 ```
 
 This is will output something like the following:
-```bash
+
+ ```bash
```

Comment:
There should not be a space before the first tick-mark (`), otherwise markdown doesn't render the syntax...